newrelic/structures_newrelic_synthetics_simple_browser_monitor.go

```diff
@@ -47,6 +47,12 @@ func buildSyntheticsSimpleBrowserMonitor(d *schema.ResourceData) (synthetics.Syn
 		simpleBrowserMonitorInput.AdvancedOptions.UseTlsValidation = &vs
 	}
 
+	typedBrowsers := expandSyntheticsBrowsers(d.Get("browsers").(*schema.Set).List())
```

Comment:
maybe `typedBrowsers` / `typedDevices` might instead be better referred to as `browsersValuesAsList` / `devicesValuesAsList`? Just sharing a thought for us to think about 😄 not to have this changed right away - let's just think about it

Reply:
I feel typedBrowsers is more appropriate, because we are type casting a already list type data to user defined data type.

No Offence 😅
<img width="745" alt="Screenshot 2024-09-10 at 7 59 51 PM" src="https://github.com/user-attachments/assets/8883614a-3372-45b5-9ad3-1b48942da422">